r/R/dplyr-datetime-helpers.R

```diff
@@ -159,6 +159,21 @@ build_formats <- function(orders) {
   orders <- gsub("[^A-Za-z_]", "", orders)
   orders <- gsub("Y", "y", orders)
 
+  short_orders <- c("ym", "my")
+
+  if (any(orders %in% short_orders)) {
```

Comment:
Please can you add a brief comment summarising what this block of code is doing?